common/transport/http/package.json

```diff
@@ -23,16 +23,16 @@
   "scripts": {
     "lint": "tslint --project . -c ../../../tslint.json",
     "build": "tsc",
-    "unittest-min": "istanbul cover --report none ../../../node_modules/mocha/bin/_mocha -- --reporter dot test/_*_test.js",
-    "alltest-min": "istanbul cover --report none ../../../node_modules/mocha/bin/_mocha -- --reporter dot test/_*_test*.js",
-    "unittest": "istanbul cover ../../../node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test.js",
-    "alltest": "istanbul cover ../../../node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test*.js",
-    "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min && npm -s run check-cover",
+    "unittest-min": "nyc --reporter=lcov --report none ../../../node_modules/mocha/bin/_mocha -- dot test/_*_test.js",
+    "alltest-min": "nyc --reporter=lcov --report none ../../../node_modules/mocha/bin/_mocha -- dot test/_*_test*.js",
+    "unittest": "nyc --reporter=lcov --reporter=text ../../../node_modules/mocha/bin/_mocha -- spec test/_*_test.js",
+    "alltest": "nyc --reporter=lcov --reporter=text ../../../node_modules/mocha/bin/_mocha -- spec test/_*_test*.js",
+    "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min ",
     "test": "npm -s run lint && npm -s run build && npm -s run unittest",
     "check-cover": "istanbul check-coverage --statements 94 --branches 87 --functions 100 --lines 94"
```

Comment:
>"check-cover": "istanbul check-coverage --statements 94 --branches 87 --functions 100 --lines 94" [](start = 4, length = 97)

The changes in this file are significantly less thant in core and amqp.  Why?

Reply:
I'm sorry I'm halfway through editing these. Give me a second to finalize the PR.